app/helpers/proposal_fields_helper.rb

```diff
@@ -8,11 +8,22 @@ def proposal_field_options(field)
       []
     else
       options = []
-      text_value = %w[value text]
-      field.options.each do |k, v|
-        options.push(v) if text_value.include?(k)
+      field.options.each do |option|
+        options.push([option.last["text"], option.last["value"]])
       end
-      [options]
+      options
+    end
+  end
+
+  def options(field)
+    if field.options == "{}"
+      []
+    else
+      opt = []
+      field.options.each do |option|
+        opt.push(option.last['text'])
+      end
+      opt
```

Comment:
Just a suggestion for these situations, you could remove 2 lines with:
```
field.options.each_with_object([]) do |option, opt|
  opt.push(option.last['text'])
end
```
I believe this loop returns the last assignment, `opt`, so no need to specify it after.